tests/oc_template_test.go

```diff
@@ -0,0 +1,257 @@
+
+	vmTemplate := `{
+		"kind": "Template",
+		"objects": [
+			{
+				"kind": "OfflineVirtualMachine",
+				"spec": {
+					"template": {
+						"spec": {
+							"domain": {
+								"resources": {
+									"requests": {
+										"memory": "${{MEMORY}}"
+									}
+								},
+								"cpu": {
+									"cores": "${{CPU_CORES}}"
+								},
+								"devices": {
+									"disks": [
+										{
+											"volumeName": "registryvolume",
+											"disk": {
+												"bus": "virtio"
+											},
+											"name": "disk0"
+										},
+										{
+											"volumeName": "cloudinitvolume",
+											"disk": {
+												"bus": "virtio"
+											},
+											"name": "disk1"
+										}
+									]
+								}
+							},
+							"volumes": [
+								{
+									"name": "registryvolume",
+									"registryDisk": {
+										"image": "kubevirt/fedora-cloud-registry-disk-demo:latest"
+									}
+								},
+								{
+									"name": "cloudinitvolume",
+									"cloudInitNoCloud": {
+										"userDataBase64": "I2Nsb3VkLWNvbmZpZwpwYXNzd29yZDogYXRvbWljCnNzaF9wd2F1dGg6IFRydWUKY2hwYXNzd2Q6IHsgZXhwaXJlOiBGYWxzZSB9Cg=="
+									}
+								}
+							]
+						},
+						"metadata": {
+							"labels": {
+								"kubevirt-ovm": "ovm-${NAME}"
+							}
+						}
+					}
+				},
+				"apiVersion": "kubevirt.io/v1alpha1",
+				"metadata": {
+					"labels": {
+						"kubevirt-ovm": "ovm-${NAME}"
+					},
+					"name": "${NAME}"
+				}
+			}
+		],
+		"apiVersion": "v1",
+		"parameters": [
+			{
+				"name": "NAME",
+				"description": "Name for the new VM"
+			},
+			{
+				"name": "MEMORY",
+				"value": "4096Mi",
+				"description": "Amount of memory"
+			},
+			{
+				"name": "CPU_CORES",
+				"value": "4",
+				"description": "Amount of cores"
+			}
+		],
+		"metadata": {
+			"labels": {
+				"miq.github.io/kubevirt-is-vm-template": "true",
+				"kubevirt.io/os": "fedora27"
+			},
+			"annotations": {
+				"description": "OCP KubeVirt Fedora 27 VM template",
+				"tags": "kubevirt,ocp,template,linux"
+			},
+			"console.openshift.io/iconClass": "icon-fedora",
+			"name": "fedora-vm-template"
+		}
+	}`
+
+	generateVmJsonFromTemplate := func(vmName string, vmTemplateJsonFile string) string {
+		By("Converting offline vm template into JSON file via oc-process command")
+		out, err := tests.RunOcCommand("process", "-f", vmTemplateJsonFile, vmTemplateParams)
+		Expect(err).ToNot(HaveOccurred())
+		vmJsonFile, err := tests.WriteJson(vmName, out)
+		Expect(err).ToNot(HaveOccurred())
+		return vmJsonFile
+	}
+
+	createVmFromJson := func(vmName string, vmJsonFile string) {
+		var message = ""
+		By("Creating offline vm from JSON file via oc-create command")
+		out, err := tests.RunOcCommand("create", "-f", vmJsonFile)
+		Expect(err).ToNot(HaveOccurred())
+		message = fmt.Sprintf("offlinevirtualmachine \"%s\" created", vmName)
+		Expect(out).To(Equal(message))
+
+		Eventually(func() bool {
+			out, err := tests.RunOcCommand("get ovms")
```

Comment:
I tried RunOcCommand this function in my own tests like `RunOcCommand("get pvc")` it throw an error, but when I change to `RunOcCommand("get", "pvc")` it works, are you sure this is correct ?